rwengine/src/engine/Animator.cpp

```diff
@@ -24,21 +24,16 @@ void Animator::tick(float dt) {
         glm::quat rotation{1.0f,0.0f,0.0f,0.0f};
     };
 
-#if 0
```

Comment:
I'm removing these snippets, because it doesn't  even compile and makes code more complex.